app/views/pages/accredited_body_new_features.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">New features</h1>
+
+    <p class="govuk-body">
+      We've recently added new features to further support accredited bodies.
+    </p>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-m">Courses as an accredited body</h2>
+
+    <p class="govuk-body">
+      In this section, you can see:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>who lists you as their accredited body</li>
+      <li>which courses you're the accredited body for</li>
+    </ul>
+
+    <p class="govuk-body">
+      <a href="#" class="govuk-link">View this feature</a>
```

Comment:
Should this be linking somewhere?

Reply:
Not at the moment, it is a separate ticket